docs-source/content/userguide/cicd/_index.md

```diff
@@ -56,7 +56,7 @@ outside of the image.  The second approach puts the JDK, WebLogic binaries
 and the domain all in the Docker image.  Both of these approaches are perfectly
 valid (and fully supported) and they have various advantages and disadvantages.
 
```

Comment:
The paragraph above needs to mention the new option/approach - Molde-in-Image - too.

Reply:
@doxiao I haven't started on this document yet - just got the 'token' - and am not sure if there'll be time to get to it before RC1 code freeze.  Please hold off on reviewing.

Reply:
@tbarnes-us we already updated this doc and mentioned three approaches (see the paragraph below this comment). We probably at least want to make this doc itself consistent.

Reply:
@doxiao 

I've updated the entire CI/CD doc section via pending pull https://github.com/oracle/weblogic-kubernetes-operator/pull/1502  

Please review the pull and resolve this comment if you agree the pull covers this comment.

Reply:
I'll review the other PR and comment if needed.